contributing/documentation/building_the_manual.rst

```diff
@@ -1,17 +1,18 @@
 .. _doc_building_the_manual:
 
 Building the manual with Sphinx
-===============================
+###############################
 
 This page explains how to build a local copy of the Godot manual using the
 Sphinx docs engine. This allows you to have local HTML files and build the
 documentation as a PDF, EPUB, or LaTeX file, for example.
 
-Before you get started, make sure that you have:
-
-- `Git <https://git-scm.com/>`_
-- `make <https://www.gnu.org/software/make/>`_ (unless you’re using Windows)
-- `Python <https://www.python.org/>`_ 3
+.. important::
+    Before you get started, make sure that you have
+    `Git <https://git-scm.com/>`_,
+    `Python 3 <https://www.python.org/>`_,
+    and, unless you are using Windows,
+    `make <https://www.gnu.org/software/make/>`_.
```

Comment:
```suggestion
Before you get started, make sure that you have:

- `Git <https://git-scm.com/>`_
- `make <https://www.gnu.org/software/make/>`_ (unless you’re using Windows)
- `Python <https://www.python.org/>`_ 3
```

Reverting this, as the 2 boxes immediately one after the other _is_ a bit happy, having built and checked the page again.